sdk/identity/Azure.Identity.BrokeredAuthentication/src/InteractiveBrowserCredentialBrokerOptions.cs

```diff
@@ -23,11 +24,26 @@ public InteractiveBrowserCredentialBrokerOptions(IntPtr parentWindowHandle) : ba
             _parentWindowHandle = parentWindowHandle;
         }
 
+        /// <summary>
+        /// Creates a new instance of <see cref="InteractiveBrowserCredentialBrokerOptions"/> to configure a <see cref="InteractiveBrowserCredential"/>.
+        /// </summary>
+        /// <param name="parentWindowHandle">Handle of the parent window the system authentication broker should be docked to.</param>
+        /// <param name="enableMsaPassthrough">Enables Microsoft Account (MSA) passthough.</param>
+        public InteractiveBrowserCredentialBrokerOptions(IntPtr parentWindowHandle, bool enableMsaPassthrough) : base()
```

Comment:
Can we make this a public setable property `EnableMsaPassthrough` since it's not required? Then we wouldn't need new constructor overloads.

Reply:
Done.